### ISSUE TITLE
Feature Request: Cloak env variables in Docker definition .yaml files

### ISSUE THREAD

**Issue by akulmehta** [dotenvx/dotenv-vscode#82]
Very nice extension. It would be great if the extension could also cloak the environment variables in .yaml files.

**Comment by motdotla** [comment]
Interesting. @akulmehta, can you share an example .yaml file like that?

**Comment by motdotla** [comment]
What frameworks or languages use this .yaml approach? I'm just curious before we start work on this. Why not just use a .env file?

**Comment by akulmehta** [comment]
@motdotla .yaml is used in Docker for containerized apps and serverless apps. Docker in turn is used by several frameworks and apps for deployment.

**Comment by motdotla** [comment]
I see. You're using environment variables inside a docker yaml definition file.

Optionally, you could reference a .env file from there instead. We recommend this because it is safer and also more flexible (you can now swap in different .env files and manage them from a secrets manager like dotenv-vault or others)

```
version: '3.8'
services:
  my-service:
    image: your-image
    env_file:
      - .env
```

BUT I acknowledge this is also a common and convenient way to define them. I've done it myself many times.

We'll consider this feature as others request it as well. It would be nice to have.

**Comment by motdotla** [comment]
Going out as part of 1.1.0 release.
